tough/tests/http.rs

```diff
@@ -25,7 +25,7 @@ mod http_happy {
     /// Test that `tough` works with a healthy HTTP server.
     #[test]
     fn test_http_transport_happy_case() {
-        run_http_test(HttpTransport::new());
+        run_http_test(HttpTransportBuilder::new().build());
```

Comment:
```suggestion
        run_http_test(HttpTransport::default());
```
nit: then we wouldn't need to import Builder either.